docs/manual.typ

```diff
@@ -228,7 +228,7 @@
 		node(c(2,1,1), $T_G P times.circle T^* X$)
 
 
-		// aways
```

Comment:
This one's actually intentional, even though not a real word — "aways" as in plural of "away" for the edges going away from the page:P